vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -477,6 +474,17 @@ def build_template_grid(templates,sort_order="DESC",sort_by="name")
     @temp[:templates] = sorted.uniq
   end
 
+  def dialog_partial_for_workflow
+    case (@edit || @options).try(:[], :wf)
+    when MiqProvisionWorkflow
+      "prov_dialog"
+    when MiqHostProvisionWorkflow
```

Comment:
We shouldn't just fall through to host provisioning.  It has a Workflow class, lets use it.